draft-ietf-quic-transport.md

```diff
@@ -2013,20 +2018,20 @@ controller.  Clients are only constrained by the congestion controller.
 
 A token sent in a NEW_TOKEN frame or a Retry packet MUST be constructed in a
 way that allows the server to identify how it was provided to a client.  These
-tokens are carried in the same field, but require different handling from
+tokens are carried in the same field but require different handling from
 servers.
 
 
-### Address Validation using Retry Packets {#validate-retry}
+### Address Validation Using Retry Packets {#validate-retry}
```

Comment:
`Using` should be capitalized in a section title, but `during` (above) shouldn't? Just curious, are there "official" rules on which words to capitalize?

Reply:
I'm surprised to see "during" not be capitalized.

Reply:
So was I.  I will ask directly and confirm that this was deliberate.